src/std/crypto/libcrypto.ss

```diff
@@ -582,14 +604,14 @@ static int ffi_EVP_DigestVerify(EVP_MD_CTX *ctx, ___SCMOBJ sig, ___SCMOBJ tbs)
   return EVP_DigestVerify(ctx, U8_DATA(sig), U8_LEN(sig), U8_DATA(tbs), U8_LEN(tbs));
 }
 
-static int ffi_EVP_DigestSignInit(EVP_MD_CTX *ctx, EVP_PKEY *pkey)
+static int ffi_EVP_DigestSignInit(EVP_MD_CTX *ctx, EVP_MD *type, EVP_PKEY *pkey)
 {
- return EVP_DigestSignInit(ctx, NULL, NULL, NULL, pkey);
+ return EVP_DigestSignInit(ctx, NULL, type, NULL, pkey);
 }
 
-static int ffi_EVP_DigestVerifyInit(EVP_MD_CTX *ctx, EVP_PKEY *pkey)
+static int ffi_EVP_DigestVerifyInit(EVP_MD_CTX *ctx,const EVP_MD *model, ENGINE *e, EVP_PKEY *pkey)
```

Comment:
engine is fucking useless and afaik it is getting deprecated, so please don't add this argument. It should be just NULL.